nnoir-onnx/nnoir_onnx/operators/mul.py

```diff
@@ -0,0 +1,19 @@
+from nnoir.functions import *
+from .utils import *
+
+
+class OpMul(Op):
+
+    def __init__(self, node, *args):
+        super().__init__(node, *args)
+
+    def to_function(self, env, constants):
+        [a, b] = self.node.input
+        if a in constants and b not in constants:
+            return [MulConstant([b], list(self.node.output), value=encode_ndarray(constants[a]))]
+        elif a not in constants and b in constants:
+            return [MulConstant([a], list(self.node.output), value=encode_ndarray(constants[b]))]
```

Comment:
MulConstant is operation for (dynamic)tensor * (constant)scalar. Scale operation is suitable for this (dynamic)tensor * (constant)tensor case. (This distinction is from historical one)